frontend/src/components/common/ApiVersionInfo.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ApiVersionInfo } from './ApiVersionInfo';
+import renderer, { act } from 'react-test-renderer';
+
+const defaultVersion = {
+  environment: 'test',
+  version: '11.1.1.1',
+};
+
+const mockGetVersion = jest.fn(async () => {
+  return Promise.resolve({ data: defaultVersion });
+});
+
+jest.mock('hooks/pims-api', () => ({
+  useApiHealth: () => ({
+    getVersion: mockGetVersion,
+  }),
+}));
+
+describe('ApiVersionInfo suite', () => {
+  afterEach(() => {
+    mockGetVersion.mockClear();
+  });
+
+  it('Displays version component', async () => {
+    await act(async () => {
+      const result = renderer.create(<ApiVersionInfo />).toJSON();
+      expect(result).toMatchSnapshot();
+    });
+  });
+
+  it('Displays version information', async () => {
+    await act(async () => {
+      const { findByTestId } = render(<ApiVersionInfo />);
+      const element = await findByTestId('version');
+      expect(element).toContainHTML(`v${defaultVersion.version}`);
+      expect(mockGetVersion).toHaveBeenCalledTimes(2);
```

Comment:
This shouldn't have been called 2 times.  But for some reason `useApi` is, even though it has empty array `[]`.